_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+---
+title: The Western Grid, Explained
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2019-02-24 08:00:00
+categories:
+  - Featured
+tags:
+  - plss
+  - history
+---
+
+<figure class="caption caption--right">
+  <a href="{{ "/images/SaltLakeHistoric.jpg" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/SaltLakeHistoric_sm.jpg" | prepend: site.baseurl }}" alt="1891 Salt Lake City Bird's Eye View">
+  </a>
+  <figcaption class="caption__text">Fig 1: 1891 Salt Lake City Bird's Eye View</figcaption>
```

Comment:
Also, @steveoh I'll leave this to you because I have no idea how to do it. Figure 1 should be footnote 1, and the footnote should be: 

American Publishing Company, “Salt Lake City 1891 Bird's Eye View,” Historic Map Works, Residential Genealogy, accessed March 1, 2019. http://www.historicmapworks.com/Atlas/US/9286/Salt+Lake+City+1891+Bird%27s+Eye+View/.

Reply:
a quick search through kramdown shows how to do it. https://kramdown.gettalong.org/quickref.html#footnotes